### DISCUSSION TITLE
Attacker Src IP Reputation

### DISCUSSION THREAD

**Discussion by cagriaslan** [telekom-security/tpotce#961] [Q&A] [answered]
Hi all,
I wonder how tpot get this information. Does anyone know?
Regards

**Comment by t3chn0m4g3** [comment, marked as answer]
It's a kind of [magic](https://github.com/telekom-security/tpotce/blob/5f29516197f80369e72c84b581da369bf840dbba/docker/elk/logstash/dist/update.sh#L24-L36), source [here](https://github.com/telekom-security/listbot).
